ext/standaloneusers/xml/schema/CRM/Standaloneusers/User.xml

```diff
@@ -199,5 +202,8 @@
     <type>varchar</type>
     <length>40</length>
     <readonly>true</readonly>
+    <permission>
+      <or>administer CiviCRM, cms:administer users</or>
```

Comment:
I feel like this should be further restricted since "administer CiviCRM" tends to be given out willy-nilly.
If we tighten it initially it could always be loosened later:
```suggestion
      <or>all CiviCRM permissions and ACLs, cms:administer users</or>
```

Actually... that might be redundant. You don't have to declare the super-duper "all CiviCRM permissions and ACLs" permission, it just is...

So it would be 
```suggestion
      cms:administer users
```